sonar-scanner-engine/src/main/java/org/sonar/scanner/report/ReportPublisher.java

```diff
@@ -178,7 +181,11 @@ String upload(File report) {
       post.setParam("characteristic", "incremental=true");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "characteristic" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)